tools/docs/manager.ts

```diff
@@ -113,19 +113,36 @@ export async function getManagersGitHubIssues(): Promise<
 export async function generateManagers(dist: string): Promise<void> {
   const managers = getManagers();
   const managerIssuesMap = await getManagersGitHubIssues();
-  const allLanguages: Record<string, string[]> = {};
+  const allCategories: Record<string, string[]> = {};
   for (const [manager, definition] of managers) {
-    const language = definition.language ?? 'other';
-    allLanguages[language] = allLanguages[language] || [];
-    allLanguages[language].push(manager);
     const { defaultConfig, supportedDatasources } = definition;
     const { fileMatch } = defaultConfig as RenovateConfig;
     const displayName = getDisplayName(manager, definition);
+
+    const categories = definition.categories ?? ['other'];
```

Comment:
```suggestion
    const categories = definition.categories ?? [];
```
otherwise this can imply users can use `others` as filter value. we need to ensure that this is clear.